variables.tf

```diff
@@ -10,6 +10,18 @@ variable "name" {
   default     = ""
 }
 
+variable "database_subnet_group_name" {
+  description = "Name to be used on DB Subnet Group resource as identifier"
```

Comment:
It would be nice to add rules for the DB subnet group name.
`Must contain from 1 to 255 characters. Alphanumeric characters, spaces, hyphens, underscores, and periods are allowed.`

Reply:
agreed - I think this would be helpful for users to know if there is criteria that must be followed